source/apickli/apickli.js

```diff
@@ -83,6 +83,7 @@ function Apickli(scheme, domain, fixturesDirectory, variableChar) {
     this.fixturesDirectory = (fixturesDirectory ? fixturesDirectory : '');
     this.queryParameters = {};
     this.formParameters = {};
+    this.predefinedOptions = {};
```

Comment:
I like this idea however we should make it super clear that these are the parameters for the request library only. I think if we rename this variable this is a great thing to add

Reply:
@seantdg Ok, I'll rename it as requestLibOptions and add it to my branch for pull request. Would it look better?